app/src/componentLibrary/modal/modalLayout/modalLayout.jsx

```diff
@@ -67,38 +67,37 @@ export const ModalLayout = ({
   useOnClickOutside(modalRef, allowCloseOutside && closeModal);
 
   return (
-    <div
-      className={cx('modal-layout', { [`overlay-${overlay}`]: overlay })}
-      data-automation-id="modalWindow"
-    >
-      <div className={cx('scrolling-content')}>
-        <Scrollbars>
-          <CSSTransition
-            timeout={300}
-            in={isShown}
-            classNames={cx('modal-window-animation')}
-            onExited={onClose}
-          >
-            {(status) => (
-              <div
+    <AnimatePresence onExitComplete={onClose}>
+      {isShown && (
+        <div
+          className={cx('modal-layout', { [`overlay-${overlay}`]: overlay })}
+          data-automation-id="modalWindow"
+        >
+          <div className={cx('scrolling-content')}>
+            <Scrollbars>
+              <motion.div
                 className={cx('modal-window', { [`size-${modalSize}`]: modalSize }, className)}
                 ref={modalRef}
                 tabIndex="0"
+                initial={{ opacity: 0, marginTop: '-100px' }}
```

Comment:
Please, add key property here. 
Direct children must each have a unique key prop so AnimatePresence can track their presence in the tree.